src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx

```diff
@@ -110,60 +99,69 @@ const ReleaseFootnotesSection = ({
   return (
     <>
       <h2>Footnotes</h2>
-      {!hasSufficientData && (
-        <p>
-          Before footnotes can be created, relevant data files need to be
-          uploaded. That can be done in the{' '}
-          <Link to="#data-upload">Data uploads section</Link>.
-        </p>
-      )}
-      {loading && <LoadingSpinner />}
-      {!loading && hasSufficientData && footnoteMeta && footnoteMetaGetters && (
-        <>
-          {canUpdateRelease && (
-            <FootnoteForm
-              {...footnoteForm}
-              footnote={undefined}
-              onOpen={footnoteFormControls.create}
-              onCancel={footnoteFormControls.cancel}
-              onSubmit={footnoteFormControls.save}
-              isFirst={footnotes && footnotes.length === 0}
-              footnoteMeta={footnoteMeta}
-              footnoteMetaGetters={footnoteMetaGetters}
-            />
-          )}
-          {footnoteMeta && (
-            <>
-              <FootnotesList
-                footnotes={footnotes}
-                footnoteMeta={footnoteMeta}
-                footnoteMetaGetters={footnoteMetaGetters}
-                footnoteFormControls={footnoteFormControls}
-                canUpdateRelease={canUpdateRelease}
+      {footnotesData === undefined ||
+        (!footnotesData.hasSufficientData && (
```

Comment:
This would render momentarily whilst we're waiting the request to actual load, which seems undesirable. It should only render if there insufficient data AND it has finished loading.